src/device_trezor/device_trezor_base.cpp

```diff
@@ -252,6 +255,29 @@ namespace trezor {
       }
     }
 
+    void device_trezor_base::set_wallet_code(const std::string & wallet_code){
```

Comment:
I was struggling with the naming here, its BIP-32 wallet derivation path, `--hw-device-wallet-code 0'/0'`. I wanted to avoid using path in the simplewallet so people don't misinterpret it as file system path. 

Maybe `wallet_dpath` as "derivation path" would be better?
`--hw-device-wallet-dpath 0'/0'`  ?

Reply:
I don't know, I'm not familiar with this, your pick will be better than mine :)